src/app/zaken/zakenRequestHandler.ts

```diff
@@ -25,58 +29,104 @@ export class ZakenRequestHandler {
     });
   }
 
-  async handleRequest(cookies: string, zaakConnectorId?: string, zaak?: string, file?: string ) {
+  async handleRequest(params: eventParams) {
     // do session stuff here
-    let session = new Session(cookies, this.dynamoDBClient);
+    let session = new Session(params.cookies, this.dynamoDBClient);
     await session.init();
     if (session.isLoggedIn() != true) {
       return Response.redirect('/login');
     }
 
-    if (!zaak) {
-      return this.list(session);
+    if (!params.zaak) {
+      return this.list(session, params);
     }
 
-    if (zaak && zaakConnectorId && !file) {
-      return this.get(zaakConnectorId, zaak, session);
+    if (params.zaak && params.zaakConnectorId && !params.file) {
+      return this.get(params.zaakConnectorId, params.zaak, session);
     }
 
-    if (zaak && zaakConnectorId && file) {
-      return this.download(zaakConnectorId, zaak, file, session);
+    if (params.zaak && params.zaakConnectorId && params.file) {
+      return this.download(params.zaakConnectorId, params.zaak, params.file, session);
     }
     return Response.error(400);
   }
 
-  async list(session: Session) {
+  async list(session: Session, params: eventParams) {
     const user = UserFromSession(session);
 
     const endpoint = 'zaken';
-    let zaakSummaries;
+    let zakenList;
     let timeout = false;
     try {
+      if (params.xsrfToken) {
```

Comment:
Waarom is dit gebaseerd op het xsrf token?

Reply:
Ah, die is nog oud. Had ik op de accept-header willen doen. Logica (die ik toe zal voegen): De frontend-response heeft al feedback, en is de fallback, dus dan mag het langer duren.